src/components/minimap/styles/minimap.scss

```diff
@@ -0,0 +1,45 @@
+@import './node';
+
+.pipeline-minimap__graph {
+  display: block;
+}
+
+.pipeline-minimap-container {
+  $offsetBottom: 2px; /* Sub-pixel adjustment */
+  position: absolute;
+  top: calc(100% + #{$offsetBottom});
+  left: 100%;
+  width: 500px;
+  height: 222px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.07);
```

Comment:
I see what you mean, though I do think for a minimap it's a good idea for it to be relatively unobtrusive as otherwise it's tough to focus on the main screen.

We should check with @GabrielComymQB on this, but I think if we did want to separate it more then maybe just a subtle pixel border might be cleaner than a shadow? I think potentially in the designs there might not be any shadow actually, I was just naturally following from the sidebar but that might be the intended to keep a crisp contrast?

Reply:
The CSS I supplied is definitely subtle - it's only 2px and virtually invisible:
![image](https://user-images.githubusercontent.com/1155816/88186363-fa687b00-cc2c-11ea-8374-2ec98eb657d8.png) 
![image](https://user-images.githubusercontent.com/1155816/88186452-1835e000-cc2d-11ea-9874-8e9161f62527.png)

But you could definitely try changing it to `0 0 0 1px`, I just thought this was subtler

Reply:
Yeah I'd tried it, that drop shadow is subtler but since it's still soft it doesn't give a crisp edge which I think might be important for contrast?

Reply:
Happy to yield to @GabrielComymQB here, but my main point is that I think it should be a white glow/outline rather than a black shadow, to make it easier to identify the edge of the box.